SerialRNTupleSource.h

```diff
@@ -46,7 +47,7 @@ namespace cce::tf {
   private:
     void fillViews(ROOT::Experimental::RNTupleReader& iReader, std::vector<std::string> const& iFieldIDs);
     SerialTaskQueue* queue_;
-    std::vector<ROOT::Experimental::RNTupleView<void>> views_;
+    std::vector<ROOT::Experimental::RNTupleView<void, false>> views_;
```

Comment:
The new declaration is: template `template <typename T, bool UserProvidedAddress> RNTupleView`.   Is the mimicking framework providing to RNtuple or getting from RNTuple the object addresses?